webapp/packages/core-blocks/src/FormControls/InputFieldNew.tsx

```diff
@@ -111,21 +134,33 @@ export const InputFieldNew: InputFieldType = observer(function InputFieldNew({
     value = mapState(value);
   }
 
+  const showRevealPasswordButton = rest.type === 'password' && !rest.disabled && !rest.readOnly;
```

Comment:
seems show password button should be shown as disabled for disabled input, and we don't want to show show password button for read-only fields